Microsoft.Identity.Web/HttpContextExtensions.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Microsoft.AspNetCore.Http;
+using System.IdentityModel.Tokens.Jwt;
+
+namespace Microsoft.Identity.Web
+{
+    public static class HttpContextExtensions
```

Comment:
>HttpContextExtensions [](start = 24, length = 21)

Isn't this JwtBearerOptions.SaveToken() doing the same thing ([https://stackoverflow.com/questions/57057749/what-is-the-purpose-of-jwtbeareroptions-savetoken-property-in-asp-net-core-2-0](https://stackoverflow.com/questions/57057749/what-is-the-purpose-of-jwtbeareroptions-savetoken-property-in-asp-net-core-2-0)) ?

Reply:
In the code I proposed, we save the decoded JwtSecurityToken, not the raw token.